addons/antennas/CfgVehicles.hpp

```diff
@@ -2,6 +2,6 @@ class CfgVehicles {
     class Land_Communication_F;
 
     class TFAR_Land_Communication_F : Land_Communication_F {
-        displayName = "Communication Tower (TFAR)";
+        displayName = "$STR_TFAR_Veh_CommunicationTower";
```

Comment:
@dedmen 
Him I have some question.
According official website (https://community.bistudio.com/wiki/Stringtable.xml)

It seen when use stringtable don't have to use quotation mark ".
But I reference some mod config.cpp or .hpp, the stringtable all have quotation mark .

So is:
1. displayName = "$STR_TFAR_Veh_CommunicationTower";
2. displayName = $STR_TFAR_Veh_CommunicationTower;

Which is right ? Thx

Reply:
This is a side effect of the binarizer.. it stringifies everything if it doesn't know what it is.. 
Stay with 1.

Reply:
Roger that.